src/style.css

```diff
@@ -1,3 +1,87 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
```

Comment:
The `@font-face` rule is not properly implemented. The `src` property should point to a font file, not a URL of a webpage. You should download the font files (usually .woff or .woff2 format) and host them in your project, or use a link provided by Google Fonts or other font service.